packages/pwa/app/pages/home/index.jsx

```diff
@@ -143,13 +143,13 @@ const Home = ({productSearchResult, isLoading}) => {
                                 'This section contains content from the catalog. {docLink} on how to replace it.',
                             id: 'home.description.shop_products',
                             description:
-                                '{docLink} is a html button that links the user to https://sfdc.co/business-manager-manage-catalgos'
+                                '{docLink} is a html button that links the user to https://sfdc.co/business-manager-manage-catalogs'
```

Comment:
This needs to be updated to the translation files. Please run `npm run build-translations` to extract and compile messages

Reply:
OK, I ran the script and updated the `en-GB.json` file.

Reply:
I also removed the formatting changes from `en-GB.json`. Are we good?